### PR TITLE
airbyte-ci: give explicit file ownership on manifest-only connector build

### PR DESCRIPTION
**Pull request by alafanechere**
## What
This change ensures that the image current user has permission to read the files we mount to the image.

### CODE REVIEW

**Comment by alafanechere** [comment]
* **#48836** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/48836?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/48836?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/common.py
Comment (alafanechere) [inline airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/common.py:1]: This was originally in the python specific build class; moved them to the base build class as it's pretty generic.

**Review by aaronsteers — APPROVED** [review]
LGTM 👍

**Review by ChristoGrab — APPROVED** [review]
@alafanechere It looks like tests are passing in CI! Given this, I'm happy to chalk up my failures as a local problem, which shouldn't be blocking

**Pull request merged**